test/integration/targets/async/tasks/main.yml

```
@@ -94,7 +94,10 @@
 - name: assert task was successfully started
   assert:
     that:
+    - fnf_task.started
     - fnf_task.started == 1
+    - fnf_task.started == True
+    - "(fnf_task.started |type_debug) == 'bool'"
     - fnf_task is started
```

This can be simplified:

```suggestion
    - fnf_task.started
```

---

yes, but the whole point is to show that the different ways of doing this still work

---

The extra assertions are redundant, since `1 == True`, `assert` verifies the value is a bool, and the `test_core` tests verify the `started` test.